dockstore-common/src/main/java/io/dockstore/common/yaml/Workflowish.java

```diff
@@ -17,6 +17,7 @@ public interface Workflowish {
     Filters getFilters();
     List<String> getTestParameterFiles();
     String getPrimaryDescriptorPath();
+    Boolean getDisableDoiGeneration();
```

Comment:
Should this be `boolean`, do we want to allow the possibility for null here?

Reply:
Boolean reflects what it is in the YAML, either a value of true or the property doesn't exist.  But we could make it a boolean here and have the implementations map the Boolean property to a boolean in the interface.